.github/workflows/image-base.yml

```diff
@@ -24,15 +27,29 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
-          file: {context}/build/Dockerfile.base
+          file: ./build/Dockerfile.base
           platforms: linux/amd64
           push: true
           tags: quay.io/sustainable_computing_io/kepler_base:latest-amd64
       - name: Build and push s390x
         uses: docker/build-push-action@v3
         with:
           context: .
-          file: {context}/build/Dockerfile.base.s390x
+          file: ./build/Dockerfile.base.s390x
           platforms: linux/s390x
           push: true
-          tags: quay.io/sustainable_computing_io/kepler_base:latest-s390x
\ No newline at end of file
+          tags: quay.io/sustainable_computing_io/kepler_base:latest-s390x
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: ./build/Dockerfile.base.arm64
+          platforms: linux/arm64
+          push: true
+          tags: quay.io/sustainable_computing_io/kepler_base:latest-arm64
+      - name: Create and upload multi-arch image
+        run: |
+            docker pull --platform=linux/s390x quay.io/sustainable_computing_io/kepler_base:latest-s390x
```

Comment:
Can this be embedded in a script and called out by `Makefile`? That'll help a local dev build.

Reply:
@rootfs done